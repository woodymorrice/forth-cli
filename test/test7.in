."hello" 2 2 substring .