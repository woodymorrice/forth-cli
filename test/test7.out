hello
2.0000 hello
2.0000 2.0000 hello
l
l
stack empty
